requirements.txt

```diff
@@ -3,3 +3,4 @@ Appium-Python-Client==0.24
 coverage==4.2
 mock==2.0.0
 nose==1.3.7
+toml
```

Comment:
Seems that not working. Module not found
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/root/src/app.py", line 9, in <module>
    import toml
ModuleNotFoundError: No module named 'toml'
```

Reply:
@SHELA I don't understand, where did you see this error? Did you run `pip install -r requirements.txt` first?

Reply:
@LerryAlexander inside docker error in log: /var/log/supervisor/docker-android.stderr.log
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/root/src/app.py", line 9, in <module>
    import toml